os/board/rtl8721csm/src/component/common/file_system/ftl/ftl.c

```diff
@@ -1554,6 +1682,12 @@ uint32_t ftl_init(uint32_t u32PageStartAddr, uint8_t pagenum)
     ftl_recover_from_power_lost();
     g_free_page_count = ftl_get_free_page_count();
 
+#if defined(CONFIG_AMEBAD_TRUSTZONE) && (FTL_SECURE == 1)
+	if (0 != ftl_secure_init()) {
+		FTL_PRINTF(FTL_LEVEL_ERROR, "ftl_secure_init fail!");

```

Comment:
I think it should be an error code here.
That's because ftl_init always returns 0 although ftl_secure_init is failed here. Is it ok?
It can bring some errors in codes that run after ftl_init.
Please check and update this.

Reply:
Added error code return and print.